core/src/test/scala/integration/kafka/server/QuorumTestHarness.scala

```diff
@@ -196,14 +199,16 @@ abstract class QuorumTestHarness extends Logging {
     }
   }
 
-  def createAndStartBroker(config: KafkaConfig,
-                           time: Time = Time.SYSTEM): KafkaBroker = {
-    implementation.createAndStartBroker(config,
-      time)
+  def createBroker(config: KafkaConfig,
+                   time: Time = Time.SYSTEM,
+                   startup: Boolean = true): KafkaBroker = {
+    implementation.createBroker(config, time, startup)
   }
 
   def shutdownZooKeeper(): Unit = asZk().shutdown()
 
+  def shutdownKRaftController(): Unit = asKRaft().shutdown()
```

Comment:
This function isn't named correctly. It will shut down more than just the controller. It also shuts down the RaftManager which is used by the brokers. If you are OK shutting down both things in your test, I'd suggest just calling `shutdown()` directly rather than creating an alias like this.

Reply:
Good point. There is no `shutdown()` method that we can invoke directly, but here we do want the RaftManager to not be shut down, so I've adjusted the method accordingly and kept the name.